src/cpuid/src/template/intel/mod.rs

```diff
@@ -50,5 +55,48 @@ pub(crate) fn msrs_to_save_by_cpuid(cpuid: &CpuId) -> HashSet<u32> {
         leaf_0x7::index0::ebx::MPX_BITINDEX,
         [MSR_IA32_BNDCFGS]
     );
+
+    // MTRR
+    cpuid_msr_dep!(
+        0x1,
+        0,
+        edx,
+        leaf_0x1::edx::MTRR_BITINDEX,
+        [
+            0x200, // IA32_MTRR_PHYSBASE0
+            0x201, // IA32_MTRR_PHYSMASK0
+            0x202, // IA32_MTRR_PHYSBASE1
+            0x203, // IA32_MTRR_PHYSMASK1
+            0x204, // IA32_MTRR_PHYSBASE2
+            0x205, // IA32_MTRR_PHYSMASK2
+            0x206, // IA32_MTRR_PHYSBASE3
+            0x207, // IA32_MTRR_PHYSMASK3
+            0x208, // IA32_MTRR_PHYSBASE4
+            0x209, // IA32_MTRR_PHYSMASK4
+            0x20a, // IA32_MTRR_PHYSBASE5
+            0x20b, // IA32_MTRR_PHYSMASK5
+            0x20c, // IA32_MTRR_PHYSBASE6
+            0x20d, // IA32_MTRR_PHYSMASK6
+            0x20e, // IA32_MTRR_PHYSBASE7
+            0x20f, // IA32_MTRR_PHYSMASK7
+            0x250, // IA32_MTRR_FIX64K_00000
+            0x258, // IA32_MTRR_FIX16K_80000
+            0x259, // IA32_MTRR_FIX16K_A0000
+            0x268, // IA32_MTRR_FIX4K_C0000
+            0x269, // IA32_MTRR_FIX4K_C8000
+            0x26a, // IA32_MTRR_FIX4K_D0000
+            0x26b, // IA32_MTRR_FIX4K_D8000
+            0x26c, // IA32_MTRR_FIX4K_E0000
+            0x26d, // IA32_MTRR_FIX4K_E8000
+            0x26e, // IA32_MTRR_FIX4K_F0000
+            0x26f, // IA32_MTRR_FIX4K_F8000
+            0x277, // IA32_PAT
+            0x2ff  // IA32_MTRR_DEF_TYPE
+        ]
+    );
+
+    // MCE
```

Comment:
It would be good to expand a bit on this comment. We want to save MCE MSRs that we define statically here. But the number of MCE banks is variable and depends on the CPU. Maybe add a note here that this will need to be changed in the future so that the information is not lost.

Reply:
done